diff --git a/adobexmp/XMP-Toolkit-SDK/XMPFiles/source/PluginHandler/OS_Utils_Linux.cpp b/adobexmp/XMP-Toolkit-SDK/XMPFiles/source/PluginHandler/OS_Utils_Linux.cpp
index 05675bc..09ff9c9 100644
--- a/adobexmp/XMP-Toolkit-SDK/XMPFiles/source/PluginHandler/OS_Utils_Linux.cpp
+++ b/adobexmp/XMP-Toolkit-SDK/XMPFiles/source/PluginHandler/OS_Utils_Linux.cpp
@@ -90,7 +90,7 @@ OS_ModuleRef LoadModule( const std::string & inModulePath, bool inOnlyResourceAc
 		}
 		else
 		{	// success !
-			result = (void*) (intptr_t) fileHandle;
+			result = (OS_ModuleRef) (intptr_t) fileHandle;
 			ResourceFileToPathMap::const_iterator iter = sMapResourceFileToPath.find(result);
 			if (iter == sMapResourceFileToPath.end())
 			{
@@ -102,7 +102,7 @@ OS_ModuleRef LoadModule( const std::string & inModulePath, bool inOnlyResourceAc
 	}
 	else
 	{
-		result = dlopen(reinterpret_cast<const char*>(inModulePath.c_str()), RTLD_LAZY/*RTLD_NOW*/);
+		result = (OS_ModuleRef) dlopen(reinterpret_cast<const char*>(inModulePath.c_str()), RTLD_LAZY/*RTLD_NOW*/);
 
 		if( !result )
 		{
